### PR TITLE
Add CentOS 8 and 9 stream preferences 

### PR DESCRIPTION
**Pull request by lyarwood**
**What this PR does / why we need it**:

WIP as I'd like https://github.com/kubevirt/common-instancetypes/pull/2 to land first.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
Preferences for CentOS 8 and 9 stream have been introduced.
Preferences for CentOS 8 have been removed given the EOL of the project on December 31st, 2021.
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,10 +176,16 @@ VirtualMachinePreference
 centos.8.desktop
 ---
 VirtualMachinePreference
-centos.9
+centos.8.stream
```
Comment (ksimon1) [inline README.md:179]: The centos 8 was EOL on December 31st, 2021 and replaced by centos 8 stream. Do you want to keep it in the list?

Comment (lyarwood) [reply README.md:179]: Ah of course sorry, I'll remove it.

**Pull request merged**